src/style.css

```diff
@@ -1,3 +1,81 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav {
+  justify-content: center;
+  align-items: center;
+}
+
+.nav__list {
+  list-style: none;
+  text-align: right;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+}
+
+.img {
+  width: 40px;
+  height: 40px;
+}
+
+.nav__link {
+  text-decoration: none;
+  color: black;
+  text-transform: uppercase;
+  height: 60px;
+  align-items: center;
+  display: flex;
+}
+
+.two,
+.three,
+.four,
+.five,
+.six,
+.seven,
+.eight {
+  margin-left: 20px;
+}
```

Comment:
<img width="963" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/70220667/a35cc17e-27af-48d9-9851-322e2009cbfa">

Reply:
you can just add the margin to the `nav__link` and remove it from the first one using the `:first-child`